exchanges/kucoin/kucoin_websocket.go

```diff
@@ -964,11 +970,9 @@ func (ku *Kucoin) processMarketSnapshot(respData []byte, instrument string) erro
 		Close:        response.Data.Close,
 		LastUpdated:  response.Data.Datetime.Time(),
 	}
-	assetEnabledPairs := ku.listOfAssetsCurrencyPairEnabledFor(pair)
-	if assetEnabledPairs[asset.Spot] && ku.AssetWebsocketSupport.IsAssetWebsocketSupported(asset.Spot) && ku.CurrencyPairs.IsAssetEnabled(asset.Spot) == nil {
+	if assetEnabledPairsSpot {
 		ku.Websocket.DataHandler <- &spotTickerPrice
-	}
-	if assetEnabledPairs[asset.Margin] && ku.AssetWebsocketSupport.IsAssetWebsocketSupported(asset.Margin) && ku.CurrencyPairs.IsAssetEnabled(asset.Margin) == nil {
+	} else if assetEnabledPairsMargin {
```

Comment:
drop and separate else if as suggested by glorious code; as it acts like one or the other.

Reply:
done